### DISCUSSION TITLE
Adding File Links in Obsidian_Anki Plugin

### DISCUSSION THREAD

**Discussion by zephaniahong** [BryanJenksCommunity/FAQ#87] [Q&A] [answered]
Hey Bryan,

I watched your video on the Obsidian_Anki Plugin and tried implementing the additional field on my anki card for the obsidian file link. I am having the issue where the obsidian file link field is not showing on the obsidian application after creating that field on anki. I have tried regenerating but to no avail. I have ensured that I toggled Add File Link option in the settings as well.

Anything idea what the issue might be?

Thank you!

p.s. Really love your Youtube content(: Recently subscribed to your channel!

**Comment by tallguyjenks** [comment, marked as answer]
send me screen shots so i can better see what you're looking at :)

**Reply by zephaniahong** [reply]
Hi Bryan, I managed to get it working after re-installing the plugin. 
Thank you!
